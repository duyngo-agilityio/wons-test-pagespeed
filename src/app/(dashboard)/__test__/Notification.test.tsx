import NotificationPage from '../notification/page';

describe('Notification page', () => {
  it('should match snapshot', () => {
    const { container } = testLibJestUtils.render(<NotificationPage />);

    expect(container).toMatchSnapshot();
  });
});
