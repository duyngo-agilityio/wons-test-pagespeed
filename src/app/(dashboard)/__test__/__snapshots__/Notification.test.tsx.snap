// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification page should match snapshot 1`] = `
<div>
  <div
    data-overlay-container="true"
  >
    <script>
      !function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&false)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}else{c.add('light')}if(e==='light'||e==='dark'||!e)d.style.colorScheme=e||'light'}catch(e){}}()
    </script>
    <div>
      <h2
        aria-level="2"
        class="font-bold font-dm-sans text-blue-800 dark:text-white text-5xl "
      >
        Notification page
      </h2>
      <p
        class="font-dm-sans text-blue-800 dark:text-white text-md mt-2"
      >
        Coming soon!
      </p>
    </div>
  </div>
</div>
`;
